### PR TITLE
Fixed warnings in RadioButtonGroup test

### PR DESCRIPTION
**Pull request by jcfilben**
#### What does this PR do?
Fixes warnings on `yarn test` for RadioButtonGroup-test.js. This PR has no effect on the overall testing coverage.

#### Where should the reviewer start?
Look at the warnings that were previously being generated from running `yarn test`. Run this command again on this PR and there won't be warnings for RadioButtonGroup-test.js.

#### What testing has been done on this PR?
Running `yarn test` to see if there were warnings for RadioButtonGroup-test.js

#### What are the relevant issues?
#4177 

#### Screenshots (if appropriate)
Screen Shot 2020-06-17 at 3 47 53 PM

#### Do the grommet docs need to be updated?
No

#### Should this PR be mentioned in the release notes?
No

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible

### CODE REVIEW

File: src/js/components/RadioButtonGroup/__tests__/RadioButtonGroup-test.js [hunk 1]
```diff
@@ -40,7 +40,13 @@ describe('RadioButtonGroup', () => {
   test('boolean options', () => {
     const component = renderer.create(
       <Grommet>
-        <RadioButtonGroup name="test" options={[true, false]} value />
+        <RadioButtonGroup
+          name="test"
+          options={[
+            { disabled: true, value: '1' },
+            { disabled: false, value: '2' },
+          ]}
```
Comment (ShimiSun) [inline src/js/components/RadioButtonGroup/__tests__/RadioButtonGroup-test.js:48]: This is changing the test. we still need to have a test an array of boolean values.

Comment (jcfilben) [reply src/js/components/RadioButtonGroup/__tests__/RadioButtonGroup-test.js:48]: Based on the RadioButtonGroup documentation https://v2.grommet.io/radiobuttongroup I'm not sure that the options props should be able to take in an array of boolean values.

Comment (ShimiSun) [reply src/js/components/RadioButtonGroup/__tests__/RadioButtonGroup-test.js:48]: I agree, but I think the intent of this test was to test a `label` that is "true" or "false".
As mentioned in this last week PR https://github.com/grommet/grommet/pull/4180.
@ericsoderberghp feel free to correct me if I'm wrong or clarify what was the test intention.

Comment (ericsoderberghp) [reply src/js/components/RadioButtonGroup/__tests__/RadioButtonGroup-test.js:48]: `options={[true, false]}` should be accepted. grommet supports it. It sounds like we need to update the documentation site to reflect that.

Comment (ericsoderberghp) [reply src/js/components/RadioButtonGroup/__tests__/RadioButtonGroup-test.js:48]: For those options, I would expect to see labels "1" and "2".

Comment (ShimiSun) [reply src/js/components/RadioButtonGroup/__tests__/RadioButtonGroup-test.js:48]: I fixed the issues raised from this thread on https://github.com/grommet/grommet/pull/4221 and https://github.com/grommet/grommet-site/pull/174.
@jcfilben please update the snapshots on this PR with its original test after PR #4221 will be merged. It should solve your issue I believe.

**Comment by jcfilben** [comment]
This PR is no longer needed after #4221 was merged and solved the problem

**Pull request closed without merging**